src/kube-runtime/src/runtime

```diff
@@ -55,12 +58,13 @@ ${PAI_RUNTIME_DIR}/precommands.sh > >(tee -a ${PAI_LOG_DIR}/runtime.pai.stdout)
 # Put verbose output to user.pai.all, stdout to user.pai.stdout, stderr to user.pai.stderr
 # execute user commands
 # priority=100
-echo "[INFO] USER COMMAND START"
+echo "[INFO] USER COMMAND START" >> ${PAI_LOG_DIR}/user.pai.stderr
 ${PAI_RUNTIME_DIR}/user.sh > >(tee -a ${PAI_LOG_DIR}/user.pai.stdout ${PAI_USER_LOG_FILE}) 2> >(tee -a ${PAI_LOG_DIR}/user.pai.stderr ${PAI_USER_LOG_FILE} >&2) &
```

Comment:
why do we need to run user command in background?

Reply:
Run user command in background is from original kube-runtime.
The purpose is user command may take long time and we may want to do something(e.g monitor user process) before wait.
Leave it here for future implementation.